app/views/job_invocations/index.html.erb

```diff
@@ -6,6 +6,7 @@
   <thead>
     <tr>
       <th><%= sort :description, :as => _('Description') %></th>
+      <th class="col-md-1"><%= _('Static Query') %></th>
```

Comment:
Because we will display this column even if the user selected "dynamic query", the name would be misleading. Maybe just "Search Query" would be a better name?